test/testinput/bufr_ncep_mtiasi.yaml

```diff
@@ -197,7 +194,13 @@ observations:
           longName: "Sensor Channel Number"
 
         # ObsValue
-        - name: "ObsValue/scaledRadiance"
-          source: variables/scaledRadiance
-          longName: "Scaled IASI Radiance"
+        - name: "ObsValue/scaledSpectralRadiance"
+          source: variables/scaledSpectralRadiance
+          longName: "Scaled IASI Spectral Radiance"
           units: "W m-2 sr-1 m"
+
```

Comment:
units: "W m-2 sr-1 m", I checked unit is "units: "W m-2 sr-1" without m. Could you please double check?